fourkitchens/quiltme
# QUILTME-128 Connect picsart

**Testing steps:**

Create a node and add an image in the original image field. Click save. Images from the API should be added to the Processed image field.

**Note:** 

Currently the batch process seems to be timing out, or else the API is not able to handle calls in rapid succession. The batch process dies after one.
